test/apps/iodemo/io_demo.cc

```diff
@@ -1869,38 +2086,60 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
+            std::cout << "  -n <connect timeout>        Timeout for connecting to the peer (or \"inf\")" << std::endl;
+            std::cout << "  -o <op1,op2,...,opN>        Comma-separated string of IO operations [read|write]" << std::endl;
+            std::cout << "                              NOTE: if using several IO operations, performance" << std::endl;
+            std::cout << "                                    measurments may be inaccurate" << std::endl;
+            std::cout << "  -d <min>:<max>              Range that should be used to get data" << std::endl;
+            std::cout << "                              size of IO payload" << std::endl;
+            std::cout << "  -b <number of buffers>      Number of offcache IO buffers" << std::endl;
+            std::cout << "  -i <iterations-count>       Number of iterations to run communication" << std::endl;
+            std::cout << "  -w <window-size>            Number of outstanding requests" << std::endl;
+            std::cout << "  -a <conn-window-size>       Number of outstanding requests per connection" << std::endl;
+            std::cout << "  -k <chunk-size>             Split the data transfer to chunks of this size" << std::endl;
+            std::cout << "  -r <io-request-size>        Size of IO request packet" << std::endl;
+            std::cout << "  -t <client timeout>         Client timeout (or \"inf\")" << std::endl;
+            std::cout << "  -c <retries>                Number of connection retries on client or " << std::endl;
+            std::cout << "                              listen retries on server" << std::endl;
+            std::cout << "                              (or \"inf\") for failure" << std::endl;
+            std::cout << "  -y <retry interval>         Retry interval" << std::endl;
+            std::cout << "  -l <client run-time limit>  Time limit to run the IO client (or \"inf\")" << std::endl;
+            std::cout << "                              Examples: -l 17.5s; -l 10m; 15.5h" << std::endl;
+            std::cout << "  -s <random seed>            Random seed to use for randomizing" << std::endl;
+            std::cout << "  -v                          Set verbose mode" << std::endl;
+            std::cout << "  -q                          Enable data integrity and transaction check" << std::endl;
+            std::cout << "  -A                          Use UCP Active Messages API (use TAG API otherwise)" << std::endl;
+            std::cout << "  -H                          Use human-readable timestamps" << std::endl;
+            std::cout << "  -P <interval>               Set report printing interval"  << std::endl;
+            std::cout << "" << std::endl;
+            std::cout << "  -q                          Validate data in buffers" << std::endl;
+#ifdef HAVE_CUDA
+            std::cout << "  -m <host|cuda|cuda-managed> Memory type to use" << std::endl;
+#else
```

Comment:
maybe we can have only the "cuda|cuda-managed" part in ifdef?
in case would like to add other memory types like rocm